tests/workload/nvidia-smoke/Makefile

```diff
@@ -0,0 +1,25 @@
+.DEFAULT_GOAL:=publish
+
+#### Variables
+REGISTRY ?=
```

Comment:
Should we use the same name as the root makefile `PUBLISH_IMAGES_REGISTRY`?

Reply:
Yeah, that makes sense. I don't really like the length of that variable name, but it's better to have consistency than less typing.